docs/source/tutorials/playing-command.rst

```diff
@@ -0,0 +1,129 @@
+=========================
+Playing with your command
+=========================
+
+Now that you have started :doc:`first-plugin`, maybe you want to write a more
+interesting command than the basic ``.hello`` one. Not that there is anything
+wrong with that command! The emoticons plugin is composed of commands like this
+one::
+
+    @plugin.command('shrug')
+    @plugin.action_command('shrugs')
+    def shrug(bot, trigger):
+        bot.say('¯\\_(ツ)_/¯')
+
+Which is one of the maintainers' favorite command to use. However, let's see
+if we can do a bit more complex than that.
+
+Greeting a user by their name
+=============================
+
+Have you noticed that a :func:`plugin_callable` takes two arguments? The first
```

Comment:
I'd xref this to <tt>:ref:`plugin-anatomy-callables`</tt> to 1) avoid the monospace format and 2) render as a friendly term instead of what looks like a function name.